cli/compose/loader/loader_test.go

```diff
@@ -582,7 +582,7 @@ volumes:
 
 func TestLoadWithInterpolationCastFull(t *testing.T) {
 	dict, err := ParseYAML([]byte(`
-version: "3.7"
+version: "3.9"
```

Comment:
Could you change this to version 3.8? This feature was added in 19.03 (through https://github.com/docker/cli/pull/1612), which has compose schema 3.8 as maximum

Changing to version 3.8 would allow us to back port this fix

Reply:
Done